src/components/ContactCard/ContactCard.jsx

```diff
@@ -28,7 +26,7 @@ const ContactCard = ({ title, contact }) => {
     <div>
       <header>
         <ContactIdentity name={contact.name} groups={[]} />
-        <HeaderActions />
+        {renderActions()}
```

Comment:
I use [`renderProps`](https://reactjs.org/docs/render-props.html#use-render-props-for-cross-cutting-concerns) for cross-cutting concerns:

here we need to render actions that are not under the responsibility of `<ContactCard />` rendering. The app is responsible to render and animate what happen in the actions bar.

Reply:
Plutôt 👍 aussi, mais comme tu dis, `ContactCard` est responsable de l'affichage de ces actions — est-ce qu'on ne devrait pas lui passer une liste d'actions / fonctions, et la `ContactCard` s'occupe de les mettre dans le DOM la ou elle veux ?

Reply:
`<ContactCard />` could choose **what to do** with the `renderActions` props. But it does not have to know **how to do** the actions rendering.

Reply:
I'm not sure I understand the difference between the two. What is there to do, except deciding where to render the actions?
But no big deal, I'll have a clearer idea once the final UI is in place.